### PR TITLE
Use correct widget name in Search setting explanation

### LINKED ISSUE #8712: Search: Clarify search widget name
This has come up a few times. Not sure if this change will help much, but maybe.

Rename "To configure search filters add the Jetpack Search widget to your sidebar." to "To configure search filters add the "Search (Jetpack)" widget to your sidebar."

### PR DESCRIPTION
**Pull request by gravityrail**
Fixes #8712

Use correct widget name in settings panel.

Also adds an underline to link styles in the "settings explanation" panels, since it was hard to distinguish links from non-linked text.

search-explanation-link-with-underline

### CODE REVIEW

**Comment by jeherve** [comment]
@simison suggested a different approach for the styling in #8733

**Comment by gravityrail** [comment]
@jeherve we feel like this is an incremental step towards #8733 that makes the smallest possible change, in the spirit of this being a code freeze :)

File: _inc/client/traffic/search.jsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ class Search extends React.Component {
 					{ this.props.getOptionValue( 'search' ) && (
 						<FormFieldset>
 							<p className="jp-form-setting-explanation">
-								{ __( 'To configure search filters add the {{link}}Jetpack Search widget to your sidebar{{/link}}.', {
+								{ __( 'To configure search filters add the {{link}}"Search (Jetpack)" widget to your sidebar{{/link}}.', {
```
Comment (zinigor) [inline _inc/client/traffic/search.jsx]: I'm sorry, but we can't change strings now. I should have mentioned it before, but the strings that went out with the code freeze have already been sent to the community translation portal and most of them have been already translated in most popular languages. If we change it now, this string will appear untranslated until we ship another release.

**Comment by gibrown** [comment]
Not a huge deal to not get this in IMO, let's just put it in master so it goes out with the next release.

**Comment by zinigor** [comment]
Totally fine merging this into master, thanks!

**Comment by zinigor** [comment]
Partially ported to `branch-5.8` in 24fd005d9.

**Pull request merged**